app/main/helpers/briefs.py

```diff
@@ -23,12 +23,22 @@ def get_brief(data_api_client, brief_id, allowed_statuses=None):
 
 
 def is_supplier_selected_for_brief(data_api_client, current_user, brief):
+    def domain(email):
+        return email.split('@')[-1]
+
+    current_user_domain = domain(current_user.email_address) \
+        if domain(current_user.email_address) not in ['gmail.com', 'bigpond.com', 'outlook.com', 'hotmail.com'] \
```

Comment:
Probably. We're making sure that this feature doesn't work for generic email hosting.